content/doc/tutorials/build-a-python-app-with-pyinstaller.adoc

```diff
@@ -352,17 +352,21 @@ stage runs successfully (with output)",width=100%]
 [source,groovy]
 ----
         stage('Deliver') {
-            agent {
-                docker {
-                    image 'cdrx/pyinstaller-linux:python2'
-                }
+            agent any
+            environment {
+                VOLUME = '$(pwd)/sources:/src'
+                IMAGE = 'cdrx/pyinstaller-linux:python2'
             }
             steps {
-                sh 'pyinstaller --onefile sources/add2vals.py'
+                dir(path: env.BUILD_ID) {
+                    unstash(name: 'compiled-results')
+                    sh "docker run -v ${VOLUME} ${IMAGE} 'pyinstaller -F add2vals.py'"
```

Comment:
Did you mean to use `--rm`?

Reply:
Thanks for catching that mistake!  Fixed in #2976